test/install.mk

```diff
@@ -129,10 +129,15 @@ install-policy:
 	kubectl wait deployments istio-policy -n ${ISTIO_NS} --for=condition=available --timeout=${WAIT_TIMEOUT}
 
 # This target should only be used in situations in which the prom operator has not already been installed in a cluster.
+install-prometheus-operator: PROM_OP_NS="prometheus-operator"
```

Comment:
Not sure it's common to name the namespace with -operator - maybe 'prometheus' is enough ?

Reply:
The default (from the packaged bundle) is to install to "default". That seems counter to the philosophy here. I have no strong feelings on the namespace.

Does the non-operator bit install to just `prometheus`?  Should we distinguish between the two approaches?

Reply:
I can look after my next test pass, but the non-operator installs into `istio-control` iirc.